cpp/net/url_fetcher.cc

```diff
@@ -245,6 +246,8 @@ void State::RequestDone(evhtp_request_t* req) {
               body_length);
   response_->body.swap(body);
 
+  VLOG(2) << *response_;
```

Comment:
Hopefully this will include logging the response header.  I assume it would as you reference response_->body above.